### DISCUSSION TITLE
Control remote heating

### DISCUSSION THREAD

**Discussion by ghost** [robbinjanssen/home-assistant-ojmicroline-thermostat#120] [Q&A] [answered]
Hi @robbinjanssen ,

I wonder if it is possible to control a remote floor heating. Is this integration using the cloud for control?

I have a floor heating in a different building than were my Home Assistant is running. I would like to be able to control it remotely.

And if this works, should it also work with the OCD5? The pricing of this one is a bit better. :-)

Thanks for your help.

**Comment by robbinjanssen** [comment, marked as answer]
Hi,

In theory, yes. This integration logs in to the cloud of ojmicroline and then peforms commands. So the device does not have to be in the local network. 

As far as i know the OCD5 won't work since it does [not have wifi](https://ojelectronics.com/floorheating/products/touch-thermostat-ocd5/), so it is unable to connect to the cloud. I think the OCD5 can only be used manually inside the room.
